zookeeper-contrib/zookeeper-contrib-zabtla/README.md

```diff
@@ -0,0 +1,71 @@
+# Zab-tla
+
+## Overview
+This project is devoted to providing formal specification and verification using TLA+ for the Zookeeper Atomic Broadcast(Zab) consensus protocol proposed in *Junqueira F P, Reed B C, Serafini M. Zab: High-performance broadcast for primary-backup systems[C]//2011 IEEE/IFIP 41st International Conference on Dependable Systems & Networks (DSN). IEEE, 2011: 245-256*.  
```

Comment:
Sounds like this PR is specifically scoped to the paper (which is kind of out of date), which is totally fine. Just want to point out that the actual ZAB running in ZooKeeper production today is ZAB 1.0 while the algorithm described in the paper is pre-1.0 and there are a few changes between the two. There is a canonical description of ZAB 1.0 here: https://cwiki.apache.org/confluence/display/ZOOKEEPER/Zab1.0. It would be more practically impactful if we have a specification checking the modern version of protocol but that can certainly be done after this exercise.

Reply:
Thank you for your suggestion! I didn't find a good manual describing Zab with new version before and this link will be very helpful to me. I will take time to learn about Zab 1.0!